orthanc/orthanc-raw/config/dicom.json

```diff
@@ -3,9 +3,25 @@
 
   "DicomModalities" : {
     "UCVNAQR" : {
```

Comment:
I wonder if we should change these key names to be primary and secondary?

Reply:
I've renamed them to `UCPRIMARYQR` and `UCSECONDARYQR`